### ISSUE TITLE
Shared block "More Options" menu: arrow navigation doesn't work on all the menu items

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6482]
Similar to #5245 / #5442

When opening the Shared Block "More Options" menu, arrow navigation works only on the first two items in the menu:

<img width="781" alt="screen shot 2018-04-28 at 15 27 09" src="https://user-images.githubusercontent.com/1682452/39397104-ec33b65a-4af9-11e8-9b9b-39564bfd58c1.png">

The solution in #5442 made all the buttons children of the menu (which is recommended also for accessibility).
